### DISCUSSION TITLE
HTTP to HTTPS redirect issues

### DISCUSSION THREAD

**Discussion by jlariviere1** [YOURLS/YOURLS#3704] [Q&A]
Just got YourLS set up; was more difficult and had more issues than I was hoping for!  I Think I have it working now, Mostly.  

If I enable redirect everything from http to https, according to the .htaccess edits mentioned in the below guide: 
https://yourls.org/docs/guide/server-configuration#redirect-everything-from-http-to-https

Then any new links I add, do not work (the 404 error).  If I test clicked on an https link (one of the yourls included default links) when https was Not enforced, from my Admin interface, it worked, and continue to work after https was enforced.  If I test click on one of them for the first time After https is enforced, it will Not work (404).  

Does anyone have some suggestions to check?  Maybe this is a bug, maybe I have something configured poorly?  The various guides, searches, etc. have been somewhat helpful in getting me to where I am now, but this is odd...  

Thanks!

**Comment by dgw** [comment]
I think the wrong comment is linked to from that page. Later discussion describes the same 404 issue you're having, and suggests a different approach: https://github.com/YOURLS/YOURLS/issues/2578#issuecomment-612451531

Links you clicked previously still work _for you_ because your browser cached them. 301 redirects are _aggressively_ cached by most browsers.

**Reply by dgw** [reply]
If you have a moment to give feedback on some updated documentation based on this discussion, take a look at YOURLS/website#179 🙂
